addons/survey/views/survey_templates.xml

```diff
@@ -3,7 +3,7 @@
 <data>
     <!-- Main survey layout -->
     <template id="survey.layout" name="Survey Layout" inherit_id="web.frontend_layout" primary="True">
-        <xpath expr="//div[@id='wrapwrap']" position="before">
+        <xpath expr="//head/*[1]" position="before">
```

Comment:
I see that in knowledge we did `<xpath expr="//header" position="before">`, would that work?

A little more simple than this one.

Thanks for looking into it!

Reply:
Let's go for `<xpath expr="//head" position="before">` then, simpler as well.